src/driver_humidity.cpp

```diff
@@ -16,21 +16,45 @@
  */
 
 #include "./driver_humidity.h"
-
 #include "./src/driver.pb.h"
-#include "matrix_hal/humidity_data.h"
+#include "matrix_hal/humidity_data.h" 
 
 namespace matrix_malos {
 
+float CPUTemp(){
+  
```

Comment:
Please check the previous comment. I wrote the C++ way of writing this function and I tested it so that you could use it. The code here is for C, not the same as C++11.